src/Tank/TankTower.cpp

```diff
@@ -11,10 +11,12 @@ constexpr auto SHOT_COOLDOWN = std::chrono::milliseconds{500};
 constexpr unsigned int MISSLE_DAMAGE = 20;
 
 TankTower::TankTower(const TankTowerTextures& textures,
-                     const std::chrono::milliseconds& shot_cooldown)
+                     const std::chrono::milliseconds& shot_cooldown,
+                     Sound shot_sound)
```

Comment:
You are always copy Sound object here. The best way to pass object that we want store as a member, is to use const reference to the object:
```
const Sound&
```